fixed an issue in generating docs for scripts (#1508)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
* Fixed an issue where **validate** suggests, with no reason, running **format** on missing mandatory keys in yml file.

> * Improved the error messages in **generate-docs** for scripts if the example was not provided 

Reviewer: ```suggestion
* Improved the error messages in **generate-docs**, if an example was not provided.
```
